### ISSUE TITLE
test annotation

### ISSUE THREAD

**Issue by hlh199603** [ucas-vg/TinyBenchmark#16]
Do you have test annotation? or do you have a test set that split a sub-set from training ？

**Comment by yinglang** [comment]
Test annotation have used for [challenge](https://competitions.codalab.org/competitions/24551#results), just like the [section](https://github.com/ucas-vg/TinyBenchmark/tree/master/tiny_benchmark#getting-started-) said. we have not split a specified val set from training set, you may need split by yourself(just simple change the annotation file will be ok.). we will try to give a suggestion split latter.
